server/templates/data.html.tera

```diff
@@ -7,6 +7,14 @@
   </div>
 {% endblock topbar_hack_end %}
 
+{% block controls %}
+  <div id="controls">
+    <div id="controls-right">
+      {% include "langselect" %}
```

Comment:
This will also require the `{% include "langselect-script" %}`.

Reply:
@sstangl It's in the custom javascript block on the bottom, is that alright? Just basing it on the pre-existing `lifter.html.tera`

Reply:
Yes